declarations_site/spotter/admin.py

```diff
@@ -1,26 +1,44 @@
 from django.contrib import admin
+from django.core.urlresolvers import reverse
+from django.utils.safestring import mark_safe
 from spotter.models import SearchTask, TaskReport, NotifySend
 
 
-class SearchTaskAdmin(admin.ModelAdmin):
+class TaskRelatedLinksMixin(object):
+    def user_link(self, obj):
+        return mark_safe('<a href="{}">{}</a>'.format(
+            reverse("admin:auth_user_change", args=(obj.user.pk,)),
+            obj.user.email
+        ))
+    user_link.short_description = 'User Link'
```

Comment:
user_link.allow_tags = True

Reply:
Deprecated since version 1.9:
In older versions, you could add an allow_tags attribute to the method to prevent auto-escaping. This attribute is deprecated as it’s safer to use format_html(), format_html_join(), or mark_safe() instead.